### ISSUE TITLE
SR

### ISSUE THREAD

**Issue by kenan0620** [dmitrysimkin/project#2]
why not use swiftR?

**Comment by dmitrysimkin** [comment]
@kenan0620 Because my project is written in Objective-C and it requires some extra work to use swift lib there. Plus  for me swiftR looks like sort of limitted, or I am wrong about that?

**Comment by kenan0620** [comment]
You can speak Chinese?




柯南
On 10/17/2017 15:08, dmitrysimkin wrote:

@kenan0620 Because my project is written in Objective-C and it requires some extra work to use swift lib there. Plus for me swiftR looks like sort of limitted, or I am wrong about that?

**Comment by kenan0620** [comment]
你是国内的吧？我在北京。你们项目是刚采用这个框架吗？

**Comment by dmitrysimkin** [comment]
No, I do not know Chinese. I tried to translate it but I didn't get what you are trying to say, sorry.

**Comment by kenan0620** [comment]
ok,my english is not good.I hope you can see what I means.Now ,you want use signalr to you project.what's your question?where you are ?

**Comment by kenan0620** [comment]
my project used siganlr object.afn3.0.i update the librayr for afn.

**Comment by dmitrysimkin** [comment]
cool! can you please share your work? it would be very nice of you!
I'm in Minsk, Belarus. We want to use signalR in my job project.
Also the reason why I cant use SwiftR because it does not support custom headers when we need this.

**Comment by dmitrysimkin** [comment]
By the way, have made a pull request to SignalR repo? A lot of people would appreciate your effort

**Comment by dmitrysimkin** [comment]
@kenan0620 did you received my last messages? if it's hard to understand me you can ask me.

**Comment by kenan0620** [comment]
oh, oh, i'm sorry。Now I'm looking at the information and I haven't received the email before.wait a moment.

**Comment by kenan0620** [comment]
What kind of contact do you use? How can I contact you in time?only email?QQ?wechat?

**Comment by kenan0620** [comment]
You can download the SignalR library locally and then import it into your project. This time you continue to import AFNetworking AFHTTPRequestOperation, AFHTTPRequestOperationManager, AFURLConnectionOperation to your project. That way you can use SignalR in the latest AFNetworking.

**Comment by dmitrysimkin** [comment]
You can write me on whatsapp, viber  +375447796142

**Comment by dmitrysimkin** [comment]
did you do it in your application? you have 2 versions of AFN in the app?

**Comment by kenan0620** [comment]
NO，wait a moment. i register a viber.

**Comment by kenan0620** [comment]
you can give me a viber app link on appstore?

**Comment by dmitrysimkin** [comment]
Viber Messenger – Text & Call by Viber Media SARL.
https://itunes.apple.com/by/app/viber-messenger-text-call/id382617920?mt=8

https://itunes.apple.com/by/app/viber-messenger-text-call/id382617920?mt=8

**Comment by kenan0620** [comment]
you can use Facebook? i dot use the viber.this app not sell in china.

**Comment by dmitrysimkin** [comment]
https://www.facebook.com/dma.cma

**Comment by kenan0620** [comment]
You can agree to my friend request.

**Comment by kenan0620** [comment]
https://www.facebook.com/coenen.hou
